source/elements/oneDAL/source/algorithms/clustering/kmeans.rst

```diff
@@ -1,3 +1,5 @@
+.. highlight:: cpp
+
 =======
 K-Means
```

Comment:
after thinking offline on the level of details : it probably makes sense to to have a detailed description of the algorithmic flow that would help quickly correlate api specifics (distance, accuracy threshold, number of iterations, objective function, convergence etc) with this flow. We can re-use the info from the Programming Guide. However, it does not contain this info for k-means
Also, what we are going to do with the processing of empty clusters? Do we enable the specific technique in oneDAL k-means?

Reply:
Added Lloyd's algorithm description, will add clarification about empty clusters in the future